op-chain-ops/cmd/celo-migrate/genesis.go

```diff
@@ -0,0 +1,130 @@
+
+func AlfajoresGenesisBlockFromConfig(config *params.ChainConfig) *core.Genesis {
+	alfajoresAlloc := &types.GenesisAlloc{}
+	alfajoresAlloc.UnmarshalJSON([]byte(alfajoresAllocJSON))
+	return &core.Genesis{
+		Config:    config,
+		Timestamp: 0x5b843511,
+		ExtraData: hexutil.MustDecode(alfajoresExtraData),
+		Alloc:     *alfajoresAlloc,
+	}
+}
```

Comment:
We need to load the extra data from the database since it has been filtered to remove the `IstanbulAggregatedSeal`.